src/components/default-page-template.less

```diff
@@ -1,19 +1,20 @@
 .DefaultPageTemplate {
+  position: relative;
   margin: 0;
   padding: 0;
   display: flex;
-  min-height: 100vh;
-  position: relative;
   flex-direction: column;
+  min-height: 100vh;
   min-width: @edu-min-page-width;
   max-width: 100vw;
 }
 
 .DefaultPageTemplate-contentArea {
+  position: relative;
   flex: 1 0 0%;
   display: flex;
   align-items: flex-start;
-  position: relative;
+  flex-direction: column;
```

Comment:
This change makes the content of "fullscreen" pages like login not vertically centered anymore, so I guess we'll have to set it back to `flex-direction: row;` in `&.DefaultPageTemplate-contentArea--fullScreen`

Reply:
Good catch 😬 Corrected.